classes/Cart.php

```diff
@@ -1092,9 +1092,11 @@ public static function cacheSomeAttributesLists($ipa_list, $id_lang)
             ORDER BY ag.`position` ASC, a.`position` ASC'
         );
 
+        $colon = Context::getContext()->getTranslator()->trans(': ', [], 'Admin.Global');
```

Comment:
@Progi1984 it's weird 😅 maybe we could use config there? It really is weird to translate something like this, especially with a space, let's add `:` in different place and we have two colons, one with and one without space :D

Reply:
@kpodemski It's weird. But I see with @Julievrz for the translation. And yes, this colon must be translated (for managing spaces before/after in differents languages).